grip/github_renderer.py

```diff
@@ -2,18 +2,18 @@
 from flask import abort, json
 
 
-def render_content(text, gfm=False, context=None,
+def render_content(text, api_url, gfm=False, context=None,
                    username=None, password=None):
     """Renders the specified markup using the GitHub API."""
     if gfm:
-        url = 'https://api.github.com/markdown'
+        url = '%s/markdown' % api_url
```

Comment:
Sure!

[ FWIW whenever I think of `.format()` I remember [this slide](https://www.dropbox.com/s/83ppa5iykqmr14z/Py2v3Hackers2013.pptx) of Guido Van Rossum's in reference to py2 vs. py3 :) ]

![d0460b8e-fbd1-11e4-90af-ea0ece9f53f4](https://cloud.githubusercontent.com/assets/52205/7667953/7f145a88-fbd4-11e4-9c9f-27f36ef983eb.png)